scripts/py_matter_idl/matter/idl/README.md

```diff
@@ -26,9 +26,13 @@ cluster AccessControl = 31 {
   // Enums and structs can be defined globally or be cluster specific.
   // IDL generation rules will take into account scoping (i.e. pick local defined
   // name first, things defined in one cluster are not visible in another).
+  //
+  // Constants allow a `spec_name` extra attribute to clarify the name source of
```

Comment:
Ideally we would have the "spec name" for _all_ IDL constructs, because anything in the IDL that does not allow recovering the "spec name" increases the likelihood that the IDL cannot be used as a codegen source of truth.

But that can be a followup.

Reply:
Right, will have to figure out a syntax eventyally.